dev/doc/changes.txt

```diff
@@ -51,6 +51,12 @@ In Constrexpr_ops:
   interpreting "(x y z:_)" as "(x:_) (y:_) (z:_)" while the second
   ones were preserving the original sharing of the type.
 
+In Nameops:
+
+  The API has been made more uniform. New combinators added in the
+  "Name" space name. Function "out_name" now fails with IsAnonymous
+  rather than with Failure "Nameops.out_name".
+
```

Comment:
> not strictly incompatibilities

I don't understand. Doesn't the pull-request remove these two functions:
- removed `Termops.pr_id`
- removed `Termops.pr_name`

Isn't that normally stuff we want to mention explicitely in `dev/doc/changes.txt`?

Did I misread the changes you made?